### PR TITLE
add riak-cs-stanchion command to switch stanchion manually

### PR DESCRIPTION
**Pull request by kuenishi**
Add a command to switch stanchion location setting. This command
assumes a use case where:
1. stanchion fails (or kill the old one, ensure killed)
2. decide a new stanchion with a brand new location (host,port)
3. put a new stanchion location via `riak-cs-stanchion switch`

Also, users can confirm stanchion setting by `show` subcommand.
I believe this will make operation in case of stanchion failure much easier
than before.

I'd like to discuss about the risk that this command may lead operational
mistake to diverse stanchion settings.

### CODE REVIEW

File: src/riak_cs_stanchion_console.erl [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        end).
+
+-define(SCRIPT_NAME, "riak-cs-stanchion").
+-include("riak_cs.hrl").
+
+%%%===================================================================
+%%% Public API
+%%%===================================================================
+
+%% @doc Kick off a gc round, unless one is already
```
Comment (reiddraper) [inline src/riak_cs_stanchion_console.erl]: copy-paste comment maybe?

Comment (kuenishi) [inline src/riak_cs_stanchion_console.erl]: eew, copied.

File: rel/files/riak-cs-stanchion [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
```
Comment (reiddraper) [inline rel/files/riak-cs-stanchion:2]: thoughts on a `set -e` for this script?

**Comment by reiddraper** [comment]
I like this. My biggest fear is that people won't understand that this is only a runtime change. It does not _persist_ the change, so if Riak CS is restarted, it will revert back to the old value. Maybe a warning will help?

**Comment by kuenishi** [comment]
fmm, that might be a problem, because people including me usually don't read warning. My idea, plus a warning, is to enable `/ping` at stanchion and let Riak CS ping stanchion when starting up the process. Anyway this should be documented strongly.

I can't wait built-in Rafter.

File: rel/files/riak-cs-stanchion (hunk 2 above)
Comment (kuenishi) [inline rel/files/riak-cs-stanchion:2]: I thought I had set this script executable. Or I messed up sth with git?

**Comment by kuenishi** [comment]
Ready for review again! English composition might be a bit better.

**Comment by kuenishi** [comment]
I'm merging this PR, but still open to additional or critical discussions on persistence and diversing stanchion settings.

**Comment by reiddraper** [comment]
Cool. Thanks @kuenishi.

**Pull request merged**